library/std/src/sys/unix/process/process_common.rs

```diff
@@ -202,6 +220,49 @@ impl Command {
         self.gid
     }
 
+    pub fn get_size(&mut self) -> io::Result<usize> {
+        use crate::mem;
+        let argv = &self.argv.0;
+        let argv_size: usize = argv.iter().map(|x| unsafe { strlen(*x) + 1 }).sum::<usize>()
```

Comment:
Oh great I forgot that the last element is a nullptr!

But seriously I would love to do this without `strlen`... like accumulating it in `add`.